package.json

```diff
@@ -7,7 +7,7 @@
     "angulartics-piwik": "^1.0.4",
     "bower": "^1.4.1",
     "cleave.js": "~0.7.15",
-    "errorhandler": "~1.3.0",
+    "errorhandler": "~1.5",
```

Comment:
> [Development-only error handler middleware](https://github.com/expressjs/errorhandler).

…and indeed used only `if (app.get('env') == 'development')`. Should be moved to `devDependencies` :)